nbnet.h

```diff
@@ -4657,7 +4666,8 @@ static int Endpoint_RegisterRPC(NBN_Endpoint *endpoint, int id, NBN_RPC_Signatur
         return NBN_ERROR;
     }
 
-    endpoint->rpcs[id] = (NBN_RPC){.id = (unsigned int) id, .signature = signature, .func = func};
+    NBN_RPC temp_rpc = {.id = (unsigned int) id, .signature = signature, .func = func};
```

Comment:
can this be renamed to simply "rpc" instead of "temp_rpc" ?

Reply:
I've renamed the variable temp_rpc to rpc as suggested. This change can be found in commit [b6773e4](https://github.com/nathhB/nbnet/pull/36/commits/b6773e4a3aaa391457976c7a7a2f7576f8923c57)